tutorials/abap-environment-gcts/abap-environment-gcts.md

```diff
@@ -1,7 +1,7 @@
 ---
 auto_validation: true
 title: Transport a Software Component Between two Systems
-description: Create and import a new software component into an SAP Cloud Platform ABAP Environment instance, add ABAP sources and export it to a new SAP Cloud Platform ABAP Environment instance.
+description: Create and import a new software component into an SAP BTP ABAP Environment instance, add ABAP sources and export it to a new SAP BTP ABAP Environment instance.
```

Comment:
Please use the full name "SAP Business Technology Platform" at least once at the beginning of your tutorial before you use the abbreviation.